docs/helpers-definitions.rst

```diff
@@ -104,6 +104,23 @@ Using definitions in tests
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 Definitions can be used as fixtures for tests in a number of different ways.
 
+With classic xunit-style setup/teardown::
```

Comment:
Maybe move this to the bottom to de-emphasise it? It's probably better to encourage the context manager version instead.

Also, maybe use `self.addCleanup(self.container.teardown)` instead of a `tearDown()` method?